src/Controls/tests/TestCases.HostApp/Issues/Issue24414.xaml.cs

```diff
@@ -1,11 +1,60 @@
+using Microsoft.Maui.Controls.Shapes;
+
 namespace Maui.Controls.Sample.Issues;
 
 [Issue(IssueTracker.Github, 24414, "Shadows not rendering as expected on Android and iOS", PlatformAffected.Android | PlatformAffected.iOS)]
 
 public partial class Issue24414 : ContentPage
 {
+	int fn = 1;
+	string[] labels = ["HELLO WORLD", "LLOHE WODRL", "OLLEH ORLWD", "LOHEL LODRW"];
+
 	public Issue24414()
 	{
 		InitializeComponent();
+		UpdateLabel();
+	}
+
+	private void OnTapGestureRecognizerTapped(object sender, EventArgs e)
```

Comment:
Could also update the sample adding another Button to remove/add the shadows at runtime?